### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/advanced/security/index.md`

### PR DESCRIPTION
**Pull request by hasansezertasan**
🌐 Add Turkish translation for `docs/tr/docs/advanced/security/index.md`

[Original File](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/advanced/security/index.md)

Discussion: #9193

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4ef8a7b654aee4c4cbc5a7d8c7aff5b9dd05b82d at: https://9144f74f.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit ae90cb3e38670601f9f5156f315ad92b10605621 at: https://282b7c10.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 048d9a31ea611e4757e33fc5a781d316d1ba4012 at: https://00687852.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4390e43aba54435c9a50276c977506527f8a20ce at: https://f38c0648.fastapitiangolo.pages.dev

**Review by OzgunCaglarArslan — APPROVED** [review]

**Review by OzgunCaglarArslan — APPROVED** [review]

File: docs/tr/docs/advanced/security/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Gelişmiş Güvenlik
+
+## Ek Özellikler
+
+[Tutorial - User Guide: Security](../../tutorial/security/index.md){.internal-link target=_blank} sayfasında ele alınanlardan farklı olarak güvenlikle ilgili bazı ek özellikler vardır.
```
Comment (hasansezertasan) [inline docs/tr/docs/advanced/security/index.md:5]:
```suggestion
[Tutorial - User Guide: Security](../../tutorial/security/index.md){.internal-link target=_blank} sayfasında ele alınanların dışında güvenlikle ilgili bazı ek özellikler vardır.
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit 46392b1a01ddebf55c5867db35d437844480185d at: https://2c7a9559.fastapitiangolo.pages.dev

**Pull request merged**